PanoptesNetClient/PanoptesNetClient/ApiClient.cs

```diff
@@ -53,30 +56,23 @@ public async Task<T> Get<T>(IRequest request)
         #endregion
 
         /// <summary>
-        /// Make a GET request for a single resource using an endpoint
+        /// Make a GET request for a list of resources
         /// </summary>
-        #region Generic GET using string
-        public async Task<T> Get<T>(string route)
+        #region Generic GET of List
+        public async Task<List<T>> GetList<T>(IRequest request) where T : IResource
         {
-            var att = (Path)Attribute.GetCustomAttribute(typeof(T), typeof(Path));
-            List<T> collection = await GetAsync<T>($"{att.Uri}/{route}");
-
-            if (collection.Count > 0)
-            {
-                return collection[0];
-            }
-            return default(T);
+            return await GetAsync<T>(request);
         }
         #endregion
 
         #region Main GET call
-        public async Task<List<T>> GetAsync<T>(string endpoint)
+        public async Task<List<T>> GetAsync<T>(IRequest request) where T:IResource
```

Comment:
Should this be private? It seems like `GetList` and `Get` just wrap this function